packages/fxa-auth-server/scripts/test-ci.sh

```diff
@@ -7,7 +7,7 @@ export NODE_ENV=dev
 export CORS_ORIGIN="http://foo,http://bar"
 
 DEFAULT_ARGS="--require esbuild-register --require tsconfig-paths/register --recursive --timeout 5000 --exit "
-if [ "$TEST_TYPE" == 'unit' ]; then GREP_TESTS="--grep #integration --invert "; fi;
+if [ "$TEST_TYPE" == 'unit' ]; then GREP_TESTS="--grep ^(?!.*#integration) "; fi;
```

Comment:
Any benefit to this verses the `--invert` flag?

Reply:
Oh, I was just tinkering with something. I would think the invert should be better... I might have to put this in draft. The changes worked locally, but as you can see the last couple test runs have failed. Unit tests should not be running integration tests, and it seemed as though they were [here](https://app.circleci.com/pipelines/github/mozilla/fxa/45531/workflows/45ffe9ff-f3ec-46b6-a14f-e39a4675c529/jobs/469374).

Reply:
Alright, I've reverted this change.